src/Http/ViewComposers/ConfigComposer.php

```diff
@@ -57,6 +58,7 @@ public function getConfig(): array
             'show_tax'                     => (bool) Rapidez::config('tax/display/type', 1),
             'grid_per_page'                => Rapidez::config('catalog/frontend/grid_per_page', 12),
             'grid_per_page_values'         => explode(',', Rapidez::config('catalog/frontend/grid_per_page_values', '12,24,36')),
+            'max_category_level'           => Cache::rememberForever('max_category_level', fn () => Category::withoutGlobalScopes()->selectRaw('MAX(level) as maxlevel')->first()->maxlevel),
```

Comment:
https://laravel.com/docs/master/eloquent#retrieving-aggregates
```suggestion
            'max_category_level'           => Cache::rememberForever('max_category_level', fn () => Category::withoutGlobalScopes()->max('level')),
```